corehq/apps/app_manager/xform.py

```diff
@@ -900,6 +900,11 @@ def get_questions(self, langs, include_triggers=False,
 
         if the xform is bad, it will raise an XFormException
 
+        :param include_triggers: When set to True will return label questions as well as regular questions
+        :param include_groups: When set will return repeats and group questions
+        :param include_translations: When set to True will return all the translations for the question
+        :param form: When included adds a hashtagValue to the dict returned
```

Comment:
Yeah, that would be a good change. I stopped using the `form` in https://github.com/dimagi/commcare-hq/pull/13989 but didn't realize it.